doc/ideas/gameplay.md

```diff
@@ -152,16 +208,80 @@ Multiplayer
+
+   - the container stores direct link to download maps from an OpenAge community servers for this tournament (no p2p-sharing of maps with different versions anymore, still possible in private games)
+
+   - container will contain also the drafting, which will become static during ongoing tournament
+     - before the tournament the civ, team and map-drafting is handled directly by the community server and stores information inside the container
+     - admin changes still possible for special use cases
+   - when starting the tournament every player will download the important information from their pre-chosen and fixed parts and everything will be pre-configured and grey (if no spontaneous changes are allowed)
+
+   - this brings the possibility to bring all the systems taking part in the tournament to one comparable and static setup (no admin RE because of forgotten MQ anymore)
+
+     - Regarding tournament settings, I would personally prefer to let the tournament host setup the game and the settings while players would just be "pulled" into the game lobby without having the ability to change the game's settings. Counter-Strike has a similar method of organizing tournament games. (_ColonelPanic_ @reddit)
+
+   - everything regarding the tournament (replays, civdrafts, etc.) should be stored in this container after the event (so you could easily import it into your OpenAge and replay games -> also see 2. Spectating/Casting-Mode)
+
+ - as an organiser of a tournament you should have the possibility to manage the whole tournament inside the OpenAge platform (not: making promo there, drafting here, hosting matches there, and so on)
+
+   - also the possibility to directly invite a player from your OpenAge-account based on characteristics (like direct-invites for 1v1-Top15 Ladder-Tournament or Teamgames)
+     - teams can connect their accounts to real clans reaching together a team elo depending on the combination of actual players (TatoH + DauT + TheViper > TatoH + DauT + Slam)
+
+ - make it easier to qualify for your tournament (e.g. as a potential player -> Open "Tournaments" in multiplayer menu -> open "Qualifiers" -> Play games with fixed tournament settings -> maybe qualify for Tournament)
+
+ - After clicking on the Tournament-Menu-Item you could have an overview of upcoming and open tournaments, with a calendar view of the next tournaments, a timeplan, the brackets and maybe even the possibility to one-click-spectate a Live-Tournament if you want (see 2. Spectating/Casting-Mode)
+
+ - depending on the architecture of the multiplayer it could be even possible to spare out all that client modeling and just integrate an basic in-game browser which connects just to an OpenAge-community-server and from there you could manage everything. To play a game the user could just click on a link from the game browser which gives data to an internal protocol like openage://<game-id>/<foo>/<bar>/<link-to-settings-file>/<...>
```

Comment:
We could do both actually. (connection through ingame server browser/direct IP and connection through a browser-based protocol)

Reply:
added